docs/general/administration/backup-and-restore.md

```diff
@@ -0,0 +1,72 @@
+   * Debian/Ubuntu packages: `sudo systemctl stop jellyfin` or `sudo service jellyfin stop`
+   * Windows Installer: Right-click the Tray app, and select "Quit"; or, in Process Manager, find the Jellyfin process and terminate it.
+   * MacOS Installer (.dmg): In Activity Monitor, find the Jellyfin process and terminate it.
+   * Portable Installs (regardless of platform): Stop the running `jellyfin` or `jellyfin.exe` program, however it was started.
+
+2. Copy your data and configuration directories to a destination of your choice. What you name the copies is up to you; personally, I like to use dated and versioned directory names e.g. `jellyfin.2024-05-01_10.8.13`. Where these files are also depends on the platform; generally these follow the [XDG Directory Specification](https://specifications.freedesktop.org/basedir-spec/latest/) for platforms that support it. For more information see [the configuration documentation](/docs/general/administration/configuration/#server-paths).
+
+   **NOTE**: These are default locations; if you've changed your data or config paths, use those instead.
+
+   * Official Docker: Wherever your `/data` and `/config` volumes are sourced from; this is set in your `docker-compose.yml` or in your `-v` options to `docker run`.
+   * LinuxServer.io Docker: Data and config are wherever your `/config` volume is sourced from; this is set in your `docker-compose.yml` or in your `-v` options to `docker run`.
+   * Debian/Ubuntu packages: Data is in `/var/lib/jellyfin` and config is in `/etc/jellyfin`; these are defined in `/etc/default/jellyfin`.
+   * RPMFusion Fedora/CentOS packages: Data is in `/var/lib/jellyfin` and config is in `/etc/jellyfin`; these are defined in `/etc/sysconfig/jellyfin`.
+   * Windows Tray/Installer (.exe): Data and config is in `C:\ProgramData\Jellyfin\Server` or `C:\Users\<Username>\AppData\Local\Jellyfin`, using `%PROGRAMDATA%` or `%LOCALAPPDATA%` respectively.
+   * MacOS Installer (.dmg): Data is stored in one of these paths; back up whichever one(s) exist: `~/.config/jellyfin/`, `~/.local/share/jellyfin/`, `~/Library/Application Support/Jellyfin/`.
+   * Portable Installs:
+      * Linux: Data is stored in `~/.local/share/jellyfin` and config in `~/.local/share/jellyfin`.
+      * Windows: Data and config is in `C:\Users\<Username>\AppData\Local\Jellyfin`, using `%LOCALAPPDATA%`.
+      * MacOS: Data is stored in these paths; back up whichever one(s) exist: `~/.config/jellyfin/`, `~/.local/share/jellyfin/`, `~/Library/Application Support/Jellyfin/`.
+
+   As an example, on Debian, you can do this with these commands to make a copy of both directories into a single target directory:
+
+   ```bash
+   sudo mkdir -p /media/backups/jellyfin.2024-10-27_10.9.11  # Or change the path wherever in your system makes sense to you
+   sudo cp -a /var/lib/jellyfin /media/backups/jellyfin.2024-10-28_10.9.11/data
+   sudo cp -a /etc/jellyfin /media/backups/jellyfin.2024-10-28_10.9.11/config
+   ```
```

Comment:
```suggestion
   ```bash
   TIMESTAMP=$(date +%Y-%m-%d_%H.%M.%S)
   sudo mkdir -p /media/backups/jellyfin.${TIMESTAMP}  # Or change the path wherever in your system makes sense to you
   sudo cp -a /var/lib/jellyfin /media/backups/jellyfin.${TIMESTAMP}/data
   sudo cp -a /etc/jellyfin /media/backups/jellyfin.${TIMESTAMP}/config
   ```
```

Not sure what's up with the diff preview :man_shrugging:

Reply:
I used a slightly different format (I believe keeping the jellyfin version in the name is imperative) but generally I do like this better.

Reply:
ah, I didn't even notice the Jellyfin version number, my brain saw modified ISO timestamp :laughing: